### DISCUSSION TITLE
Is there a list of all possible achievements?

### DISCUSSION THREAD

**Discussion by solomonshorser** [community/community#36035] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there a way to see what achievements are possible? Is there a way to see how much progress has been made towards specific achievements? This new feature feels like something found in a video game, but not as fully-formed - though I know it's still in beta so maybe more features will come later.

**Comment by backtoaldo** [comment, marked as answer]
There is a community repository where you find the list of all achievments, the repository: [GitHub Profile Achievements.](https://github.com/Schweinepriester/github-profile-achievements)
Some are not possible to catch but there are still many open.
